packages/core/src/__tests__/rum/instrumentation/ShallowObjectEqualityChecker.test.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+import { areObjectShallowEqual } from "../../../rum/instrumentation/ShallowObjectEqualityChecker";
+
+describe('areObjectValuesEqual', () => {
```

Comment:
This describe / it pattern is more common in js projects, this is something I would like to introduce in other tests in a separate PR.
I love that the GIVEN/WHEN/THEN pattern is pecified for integration tests, but I don't usually mention it for small unit tests like this if this is ok.